policyengine_canada/variables/gov/provinces/nb/tax/income/credits/tuition_amount/nb_tuition_credit_amount.py

```diff
@@ -0,0 +1,132 @@
+from policyengine_canada.model_api import *
+
+
+class nb_tuition_credit_amount(Variable):
+    value_type = float
+    entity = Person
+    label = "New Brunswick tuition credit amount"
+    definition_period = YEAR
+    defined_for: "nb_tuition_eligible"
+    reference = (
+        "https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/5004-s11/5004-s11-22e.pdf#page=1",
+        "https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/5004-c/5004-c-22e.pdf#page=1",
+    )
+
+    def formula(person, period, parameters):
+        age = person("age", period)
+        tuition = person("tuition_expenses", period)  # line 2
+        taxable_income = person("nb_taxable_income", period)  # line 4
+        tuition_income = person(
+            "nb_tuition_credit_amount_income", period
+        )  # line 5
+        p = parameters(
+            period
+        ).gov.provinces.nb.tax.income.credits.tuition_amount
+        return select(
```

Comment:
i dont thunk we need this structure, cant we just use the .calc function for the marginal rates

Reply:
just use .calc(taxable_income)